flipkart_scraper.py

```diff
@@ -0,0 +1,120 @@
+# The front end can make use of the passing parameters
+# In the dropdown list, the developer can give the Words like Laptops, Mobiles, Cameras, Headphones, Air conditioners etc
+# Also the user can define the limit of number of pages to scrap [It is used in both functions]
+def get_url(dropdownselection,limit):
+  urlss=[] 
+  if dropdownselection==1:
+    for i in range(1,limit):  #use n+1 pages for clearing index out of range error
+      laps='https://www.flipkart.com/laptops/pr?sid=6bo,b5g&otracker=categorytree&page='+str(i)
+      urlss=urlss+[laps]
+  elif dropdownselection==2:
+    for i in range(1,limit): 
+      mobs='https://www.flipkart.com/mobiles/pr?sid=tyy%2C4io&otracker=categorytree&page='+str(i)
+      urlss=urlss+[mobs]
+  elif dropdownselection==3:
+     for i in range(1,limit): 
+      cams='https://www.flipkart.com/cameras/pr?sid=jek,p31&otracker=categorytree&page='+str(i)
+      urlss=urlss+[cams]
+  elif dropdownselection==4:
+     for i in range(1,limit): 
+      hpo='https://www.flipkart.com/headphones/pr?sid=fcn&otracker=categorytree&page='+str(i)
+      urlss=urlss+[hpo]
+  elif dropdownselection==5:
+     for i in range(1,limit): 
+      acs='https://www.flipkart.com/air-conditioners/pr?sid=c54&otracker=categorytree&page='+str(i)
+      urlss=urlss+[acs]
```

Comment:
Creating a dictionary which stores all those links and then accessing the values would look better. Also, since `for i in range(1,limit)` is repeated a lot of times here. Maybe you can refactor the code so that it's used just once. (You can put the conditions inside the loop). Also, you can use `switch - case` here with a `default`. Would be more readable.